instrumentation/opensearch/opensearch-rest-1.0/javaagent/src/test/java/OpenSearchRestTest.java

```diff
@@ -72,6 +73,7 @@ static void setUpOpenSearch()
                 httpClientBuilder ->
                     httpClientBuilder
                         .setSSLContext(sslContext)
+                        .setSSLHostnameVerifier(new NoopHostnameVerifier())
```

Comment:
is this change related to non-localhost docker?

Reply:
It is, yes. The SSL cert used by the opensearch image has a hostname of "localhost", so non-localhost Docker will fail with an SSL exception due to "Invalid hostname on certificate". This line disables the hostname verification to prevent that.